### PR TITLE
pinpoint single withdraw object

### PR DESCRIPTION
**Pull request by CJNA**
One can't assume that api server will respond with key:'success'
It can only send msg, and dev need to see this msg in order to understand the issue

### CODE REVIEW

File: binance/client.py [hunk 1]
```diff
@@ -1932,7 +1932,7 @@ def get_account_status(self, **params):
 
         """
         res = self._request_withdraw_api('get', 'accountStatus.html', True, data=params)
-        if not res.get('success'):
```
Comment (AlexNik) [inline binance/client.py:1935]: I think you should use this code `if not res.get('success'):` everywere...

File: binance/client.py [hunk 2]
```diff
@@ -2279,9 +2279,7 @@ def withdraw(self, **params):
         :param asset: required
         :type asset: str
         :type address: required
-        :type address: str
```
Comment (AlexNik) [inline binance/client.py:2282]: keep this line

File: binance/client.py [hunk 3]
```diff
@@ -5508,6 +5558,14 @@ def futures_coin_get_position_mode(self, **params):
 
         """
         return self._request_futures_coin_api("get", "positionSide/dual", True, data=params)
+    
+    def fetch_all_coins_information(self, **params):
```
Comment (AlexNik) [inline binance/client.py:5562]: Copy of this function `get_all_coins_info`

**Review by AlexNik — CHANGES_REQUESTED** [review]

File: binance/client.py (hunk 3 above)
Comment (CJNA) [reply binance/client.py:5562]: hmph. I was going to bring this as separate PR. Sorry about confusion. Will do.

File: binance/client.py (hunk 1 above)
Comment (CJNA) [reply binance/client.py:1935]: Fair point. I think I've had older version where it just called not res['success'] directly. 
Will stick with that.

**Review by AlexNik — APPROVED** [review]

**Review by kimchirichie — APPROVED** [review]

**Pull request merged**